packages/amplify-migration-tests/src/migration-helpers/init.ts

```diff
@@ -0,0 +1,72 @@
+import { addCircleCITags, getCLIPath, isCI, nspawn as spawn } from 'amplify-e2e-core';
+
+const defaultSettings = {
+  name: '\r',
+  envName: 'integtest',
+  editor: '\r',
+  appType: '\r',
+  framework: '\r',
+  srcDir: '\r',
+  distDir: '\r',
+  buildCmd: '\r',
+  startCmd: '\r',
+  useProfile: '\r',
+  profileName: '\r',
+  region: process.env.CLI_REGION,
+  local: false,
+  disableAmplifyAppCreation: true,
+};
+
+export function initJSProjectWithProfile(cwd: string, settings: Object, testingWithLatestCodebase = false) {
```

Comment:
nit: This helper could exist within `amplify-e2e-core`.

Reply:
initJSProjectWithProfile already exists in `amplify-e2e-core`. I purposely moved this version to `amplify-migration-tests` because of the way testingWithLatestCodebase defaults to false by convention in other e2e helpers. I would have had to edit every single call to initJSProjectWithProfile in regular e2e tests otherwise.